### DISCUSSION TITLE
css/css-combinators/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#59] [Announcements]
# css/css-combinators/

CSS 提供了一组运算符，用于选择与特定元素有特殊位置关系的元素，例如选择子元素、直接子元素、相邻节点和直接相邻节点，接下来看一下每种运算符的作用。

https://zxuqian.cn/css/css-combinators/

**Comment by CosmosDiv** [comment]
老师你好，多次使用组合运算符的预览，是否多标红了一行。“一些文字”不应该变红。

**Reply by zxuqian** [reply]
是的，感谢指正，已修改
